compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1497,6 +1497,48 @@ static LogicalResult setRootConfig(mlir::FunctionOpInterface entryPointFn,
       entryPointFn, fftOp, tileSizes, DispatchLoweringPassPipeline::CPUDefault);
 }
 
+/// Sets the lowering configuration for dispatch region for
+/// linalg_ext.winograd.input_transform root op.
+static LogicalResult
+setRootConfig(mlir::FunctionOpInterface entryPointFn,
+              IREE::LinalgExt::WinogradInputTransformOp inputOp) {
+  assert(!getLoweringConfig(inputOp) && "expected lowering_config is not set");
+  auto outShape = inputOp.getOutputOperandType().getShape();
+  SmallVector<int64_t> distTileSizes(outShape.begin() + 2, outShape.end());
+  distTileSizes = llvm::map_to_vector(distTileSizes, [](int64_t size) {
+    return std::min<int64_t>(size, clDefaultDistTileSize);
+  });
```

Comment:
Can we try to get the distribution tile sizes from the unified method? I've spent some time to cleanup distribution logics, and finally they're converged now. It would be good to have them use the same method, if there are no specific concerns. Also, it can make `iree-llvmcpu-disable-distribution` automatically work for these new ops. E.g.,

```cpp
  SmallVector<int64_t> distTileSizes =
      getDefaultDistributedLevelTileSizes(inputOp, DistributionHeuristicConfig{});
```

Reply:
Ah, yeah. I wanted something working first, and I meant to come back to this and use `getDefaultDistributedLevelTileSizes`. Thanks for catching it.